web/cypress/integration/morecast-page.spec.ts

```diff
@@ -112,7 +112,7 @@ describe('MoreCast Page', () => {
     })
 
     it('Temp & RH Graph should be displayed', () => {
-      const checkNumOfTempMarkers = (num: number) => {
+      const checkNumOfTempDewpointMarkers = (num: number) => {
```

Comment:
I am not sure if combining these two is the right way. We only display dew points for Observations right?